README.md

```diff
@@ -29,6 +29,7 @@ For a more detailed description, and dependencies, visit: https://github.com/tra
     $ tar xf transmission-2.92.tar.xz
     $ cd transmission-2.92
     $ mkdir build
+    $ cd build
```

Comment:
this seems to be fixed in master already (but that's probably because this PR has sat unreviewed in `tr/tr` for so long)